### PR TITLE
Update AppHead.js

### PR DESCRIPTION
**Pull request by leungwaynetx**
Adding set session event for Crisp chatbot

### Screenshots
N/A

### CODE REVIEW

**Comment by conrad-vanl** [comment]
hey @leungwaynetx are you familiar with linting at all? This PR is failing right now due to a linter error. It should be easy to fix if you know how to discover the lint error. `yarn lint` should show you if you were running the code locally at all.

If not, then I can potentially figure another way to help. 

I tried adding a `lint` action to our GitHub actions so that GitHub will actually show you the lint error on this PR (and they make it super easy to fix the error) but I don't think you'll see that action run until you push at least one more commit up.

Edit: if you're not familiar with linting, it's just an automated tool we use to make sure your code follows consistent syntax/formatting rules. That way we don't end up with different blocks of code formatted differently. Looking at the error, I think you just need to space out the code you added a little differently to make the linter happy.

**Comment by leungwaynetx** [comment]
Thanks again Conrad. Sorry that I am a total noob here and using so much of your time. I really do appreciate your kind help and advice. Does the above commentary mean the linter was executed(??) upon my last commit (I tried to match the syntax to the similar line below my insert). Thanks again, Wayne

File: tvappweb/components/AppHead/AppHead.js [hunk 1]
```diff
@@ -68,6 +68,11 @@ function AppHead() {
     d.getElementsByTagName('head')[0].appendChild(s);
   }, []);
 
+  useEffect(() => {
+    // Feeding Chatbot with an initial event when website page loads.
+    window.$crisp.push(['set', 'session:event', [[['chaseoakstv:page:loaded', {}, 'red']]]]);
```
Comment (conrad-vanl) [inline tvappweb/components/AppHead/AppHead.js:73]:
```suggestion
    window.$crisp.push([
      'set',
      'session:event',
      [[['chaseoakstv:page:loaded', {}, 'red']]],
    ]);
```

@leungwaynetx FYI, our linter is wanting it to be formatted like this - it's supposed to automatically add these suggestions to the PR but for some reason that's not working, so I've added the code suggestion manually in this comment.

You should be able to accept the suggestion above and the PR checks will re-run and then we should be good to go.

**Review by conrad-vanl — CHANGES_REQUESTED** [review]

**Comment by leungwaynetx** [comment]
Thanks Conrad. I will get smarter as we go forward... I promise.

**Comment by leungwaynetx** [comment]
Commited

**Review by conrad-vanl — APPROVED** [review]

**Comment by conrad-vanl** [comment]
Hey @leungwaynetx please stop apologizing/feeling bad. You're doing great. 

There's a Preview URL available of your work here:
https://chase-oaks-apollos-83de97ou8-apollos.vercel.app

If you want to test to see that what you did is working, you can do so at that URL.

I've approved this PR based on a code review. I'll let you confirm that everything is working as you intend it to on that URL, and once you give me the 👍 I'll merge it and it'll get deployed to the live website.

**Comment by conrad-vanl** [comment]
@leungwaynetx since you're an "outside" collaborator I have to push a button on GitHub to give it permission to run its checks and deploy a testing url. I just did that for your last commit and you can see that in the status history:
Screen Shot 2022-04-29 at 8 13 00 AM

So just give me an @ mention whenever you need me to press the button so you can test your work, it's no problem.

https://chase-oaks-apollos-iiyxwnk45-apollos.vercel.app

**Comment by leungwaynetx** [comment]
@conrad-vanl , The testing was successful. Ready to merge whenever you are ok and ready to deploy as well. Thanks - Wayne

**Comment by conrad-vanl** [comment]
Merged. Should auto deploy shortly.

**Comment by leungwaynetx** [comment]
Thanks Conrad!

**Pull request merged**